static/fix-missing-vars.js

```diff
@@ -1,2 +1,3 @@
 const process = {env:{}}
-const global = {}
\ No newline at end of file
+const global = {}
+const development = process.env.NODE_ENV
```

Comment:
who uses this `development` - variable?

Reply:
React threw an error on windows without having this. I don't know why. Adding it and everything was good.

Reply:
now it is a string, but shouldn't it be a boolean?

Reply:
Nope should be a string